natlas-server/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-mkdir -p /data/{db,media}
+mkdir -p /data/{db,media,logs}
+ln -s /data/logs logs
+
+DATABASE_FILE=${SQLALCHEMY_DATABASE_URI//sqlite:\/\/\//}
+if [ ! -f "${DATABASE_FILE}" ]; then 
+  echo "[+] Creating database"
+  export FLASK_APP=./natlas-server.py
```

Comment:
Why is this export needed? The Dockerfile should set it already

Reply:
More copy pasta from adding_docker branch